app/controllers/trainees/degrees_controller.rb

```diff
@@ -55,6 +58,12 @@ def redirect_path
       end
     end
 
+    def apply_invalid_data_view
```

Comment:
Could this go into a before action only for `:new`?

Reply:
updated and not used here anymore.